### PR TITLE
process: Add github issue template

### LINKED ISSUE #589: Create an issue template
Create an `ISSUE_TEMPLATE.md` to help guide users raise issues.

For scenarios where the template is not appropriate, simply `CTRL-a delete` to zap it ;)

---

# Expected result

# Actual result 

# Details

Please provide the output of the following (either paste into this issue, or attach output as separate files)

- `sudo cc-runtime cc-env`
- `sudo docker info`
- `sudo cat /usr/share/defaults/clear-containers/configuration.toml`
- `sudo cat /etc/clear-containers/configuration.toml` (if it exists)

- If you have logging enabled
  - `sudo cat /var/lib/clear-containers/runtime/runtime.log`

### PR DESCRIPTION
**Pull request by jodh-intel**
Add a "magic" github file that will be used to pre-populate the text
box for:

    https://github.com/01org/cc-oci-runtime/issues/new

Fixes #589.

Signed-off-by: James O. D. Hunt <james.o.hunt@intel.com>

### CODE REVIEW

File: ISSUE_TEMPLATE.md [hunk 1]
```diff
@@ -0,0 +1,16 @@
+# Description of problem
+
+(remove this text and list the steps you followed)
```
Comment (grahamwhaley) [inline ISSUE_TEMPLATE.md]: I'd probably lean towards 'replace this text with...', like we have for the `cc-collect-data.sh` section.

File: ISSUE_TEMPLATE.md [hunk 2]
```diff
@@ -0,0 +1,16 @@
...
+
+(remove this text and explain what you thought would happen)
+
+# Actual result 
+
+(remove this text and explain what actually happened)
+
+---
+
+(replace this text with the output of the `cc-collect-data.sh` script, after you have
```
Comment (grahamwhaley) [inline ISSUE_TEMPLATE.md]: we could hotlink to the `cc-collect-data.sh` script as well

**Comment by grahamwhaley** [comment]
that 'no kvm module' error on Jenkins 16.04 (but not 17.04) is just odd - let me go press the rebuild button and see if it was sporadic or persists....

File: ISSUE_TEMPLATE.md (hunk 1 above)
Comment (jodh-intel) [reply ISSUE_TEMPLATE.md]: Agreed. Let me update that...

File: ISSUE_TEMPLATE.md (hunk 2 above)
Comment (jodh-intel) [reply ISSUE_TEMPLATE.md]: Good idea. Updating...

**Comment by jodh-intel** [comment]
I've raised an issue on the jenkins error: https://github.com/clearcontainers/jenkins/issues/4.

File: ISSUE_TEMPLATE.md (hunk 2 above)
Comment (jodh-intel) [reply ISSUE_TEMPLATE.md]: Ah - slight complication: https://github.com/clearcontainers/runtime/pull/606 changes the script to be auto-generated. So we can point to the unbuilt script, but not the script itself. What do you think?

File: ISSUE_TEMPLATE.md (hunk 1 above)
Comment (jodh-intel) [reply ISSUE_TEMPLATE.md]: Updated.

File: ISSUE_TEMPLATE.md (hunk 2 above)
Comment (grahamwhaley) [reply ISSUE_TEMPLATE.md]: heh. Oh. Maybe we can note something like 'that came with your installation, or is available after a build from source' or similar? But, fine if we leave as-is.

**Comment by jodh-intel** [comment]
Sigh. This PR is failing for an unrelated reason (I only added a doc file!):

```
14:24:07 • Failure [3.449 seconds]
14:24:07 run
14:24:07 /home/jenkins/go/src/github.com/clearcontainers/tests/functional/run_test.go:103
14:24:07   container
14:24:07   /home/jenkins/go/src/github.com/clearcontainers/tests/vendor/github.com/onsi/ginkgo/extensions/table/table.go:96
14:24:07     without '--pid-file' option [It]
14:24:07     /home/jenkins/go/src/github.com/clearcontainers/tests/vendor/github.com/onsi/ginkgo/extensions/table/table_entry.go:46
14:24:07 
14:24:07     Expected
14:24:07         <int>: 1
14:24:07     to equal
14:24:07         <int>: 0
14:24:07 
14:24:07     /home/jenkins/go/src/github.com/clearcontainers/tests/functional/run_test.go:94
```

**Comment by jodh-intel** [comment]
Hi @iphutch - please can you review?

**Comment by jodh-intel** [comment]
Coveralls seems to be "stuck" again; the 16.04 build (which ran successfully) triggered coveralls which created:

- https://coveralls.io/jobs/29667445

This *claims* the coverage has gone down slightly. However, this is clearly impossible given this PR changes no code and only adds a single document.

Merging.

**Pull request merged**